extension/src/cli/git/executor.test.ts

```diff
@@ -38,9 +39,10 @@ describe('GitExecutor', () => {
     } as unknown as EventEmitter<CliStarted>
   })
 
+  const cwd = standardizePath(__dirname) as string
```

Comment:
[F] Followed up and split this function `standardizePath(something) as string` is everywhere and dumb.